### PR TITLE
LPS-124049 Only prevent menu to open when clicking on toggle editable in edit mode

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
cc @p2kmgcl 🧞‍♂️

**Comment by sandrodw3** [comment]
ci:forward

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.css [hunk 1]
```diff
@@ -7,17 +7,7 @@
 	border-width: 0;
 	box-shadow: 0 1px 5px -1px rgba(0, 0, 0, 0.3);
 	display: none;
-	left: 0;
 	min-width: 240px;
-	position: absolute;
-	top: 100%;
+	position: fixed;
```
Comment (p2kmgcl) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.css:11]: Magical position

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.js [hunk 2]
```diff
@@ -27,10 +27,18 @@ function alignMenu() {
...
 	const parentRect = parent.getBoundingClientRect();
 
-	menu.style.position = 'fixed';
-	menu.style.left = `${parentRect.left}px`;
 	menu.style.top = `${toggleRect.bottom}px`;
-	menu.style.width = `${parent.getBoundingClientRect().width}px`;
+
+	if (configuration.panelType === 'mega-menu') {
+		menu.style.left = `${parentRect.left}px`;
+		menu.style.width = `${parent.getBoundingClientRect().width}px`;
```
Comment (p2kmgcl) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.js:34]: You have `parentRect` right above, don't need to `parent.getBoundingClientRect()` twice (this is expensive)

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
Sorry for blocking this, but using getBoundingClientRect too much can be _dangerous_

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/dropdown/index.js]: You are completely right, and I am completely sorry 😞

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**